modules/kinesis/src/main/resources/reference.conf

```diff
@@ -10,6 +10,7 @@ snowplow.defaults: {
         maxRecords: 1000
       }
       leaseDuration: "10 seconds"
+      maxLeasesToStealAtOneTimeFactor: 2.0
```

Comment:
Curious that this is a decimal, you can't have a fraction of a lease surely? Doesn't seem like a likely oversight - so I'm guessing there's a reason I don't see?

Reply:
This number is first multiplied by the number of processors.  It is helpful to make it a decimal, e.g. maybe we want a 8-core instance to steal 2 leases at one time, so we set this value to `0.25`.

I've been using this pattern in lots of places recently.  It's a good way to make the app auto-guess a good configuration, instead of putting the burden on the user to configure it appropriately for its vertical size.  And I consistently put `Factor` as a suffix of params that get multiplied by num processors.